### ISSUE TITLE
Address Data Verification Critical Accessibility Issues

### ISSUE THREAD

**Issue by governmentSponsored** [US-Trustee-Program/Bankruptcy-Oversight-Support-Systems#887]
- [ ] On a consolidation, select a different lead case, If you select a lead case that is a child of another consolidation, the wording of the alert message is confusing. See attached image:  !image-2024-09-11-08-39-52-304.png|thumbnail!  In this scenario, we would remove the first sentence in the image.
- [ ] When Case Not Listed is selected (or Lead Case Not Listed), add helper text below the table of cases, above the Required Field label, that says "Choose a new court and enter a case number, and the [lead] case will be selected for this Case Event automatically." (add lead for the one on Consolidations) (WCAG 3.2.2).
- [ ] Ensure each auto-search control (e.g. case not listed) has a basic accessible description that mentions that the controls will search for a case automatically. (WCAG 3.2.2).
- [ ] Ensure that the Consolidation Type radio buttons can be selected with a keyboard. Currently, neither Enter nor Space will cause an option to become selected. This is a problem with all radio buttons in CAMS (WCAG 2.1.1).
- [ ] Ensure focus indicator on "case not listed" search functions does not disappear when search executes (WCAG 2.4.7). 
- [ ] Add an input mask to all Case Number fields, with helper text that says "For example, 12-34567".
- [ ] Ensure that a focus indicator is always visible when tabbing through the Data Verification modals (WCAG 2.4.7)
- [ ] Ensure that text entry fields receive focus before the field's respective Clear button (WCAG 2.4.3).
- [ ] There is a phantom landmark in the header. "cams-banner-nav". It doesn't seem to point to an interactive component. We should make sure it's connected to the correct elements (WCAG 1.3.1).
- [ ] Event Type and Event Status, under Filters, should be headers, but currently are not (WCAG 1.3.1, 2.4.6)
- [ ] Add an accessible description to the jump links that take users to specific docket entries in Transfers and Consolidations. Currently they don't provide enough context that that's what they do. (WCAG 2.4.4)
- [ ] Ensure the filter toggles have the appropriate ARIA custom component widget roles. They are more than than simple buttons (WCAG 4.1.2). 
- [ ] Ensure the Mark As Lead toggles have the appropriate ARIA custom component widget roles. They are more than simple buttons (WCAG 4.1.2).
- [ ] Ensure the Case Event accordions have the appropriate ARIA custom component widget roles. They are more than simple buttons (WCAG 4.1.2). 
- [ ] Ensure the selected/active state of the Case Event filters can be programmatically determined (WCAG 1.3.1, WCAG 2.5.3).
- [ ] Add an accessible description to the Case Event accordions, to make it easier to tell what they do. May not be needed if ARIA widget roles are sufficient. Check with screen reader. (WCAG 1.3.1, 2.5.3).
- [ ] Add an accessible description to the Select a Court dropdowns, to specify that they expect a court district or division. (WCAG 3.3.2)
- [ ] For the Select a Court dropdown, review the ARIA design spec for Combo Boxes, and ensure the component behaves as expected. (WCAG 4.1.2, 2.1.1)
- [ ] Ensure the text area in the Reject modal has a programmatically-determinable label (it's labeled visually, we likely just have to ensure it's associated with the text area); this was a Critical issue found by Axe DevTools.
- [ ] Ensure all button labels appropriately adjust their text spacing when user need it (WCAG 1.4.12)
- [ ] Ensure the Required status of required input fields can be programmatically determined (WCAG 1.3.1)
- [ ] Ensure that the headers above the case event columns ("Court District," "Order Filed", "Event Type", and "Event Status") are either coded as table headers, or headers (likely H5's). Currently they are neither. (WCAG 1.3.1).
- [ ] Ensure instances of bold body text (e.g. Consolidation Type, Case number) in the transfer confirmation modal, consolidation confirmation modal, and rejection modal are tagged with <strong> tags. (WCAG 1.3.1)
- [ ] Ensure that the two checkboxes at the top of the Consolidation case list (so, include all and include topmost) are not too close together; they should be at least 24 CSS pixels away from each other (WCAG 2.5.8). 
- [ ] Replace all instances of the word Approve (Transfer button labels, Approved tag, success alerts) with Verify, so that we are using the same word consistently throughout our app. Verify is preferred over Approve, because USTP isn't actually approving anything, we're just making sure it's set up correctly. (WCAG 3.2.4).

**Comment by fmaddenflx** [comment]
Added to list:   The "logout" link in the banner at the top of the screen is white on all versions of the app and in production is white on light grey.  We need to change it to black.
